laalaa/apps/advisers/healthchecks.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from moj_irat.healthchecks import HealthcheckResponse, UrlHealthcheck, registry
+
+class CeleryWorkersHealthcheck(object):
+    def __init__(self, name):
+        self.name = name
+    
+    def __call__(self, *args, **kwargs):
+        from celery import Celery
+        
+        try:
+            app = Celery('laalaa')
+            stats = app.control.inspect().stats()
```

Comment:
I wonder if we could add a test for this check? Maybe port it back to the irat repo?

Reply:
Yeap, once it's merged, I'll ping the author of the irat repo about porting it back and helping with a test.